scripts/ExportScripts/export_profile_monolithic.py

```diff
@@ -0,0 +1,55 @@
+from o3de.export_project import *
+from o3de import manifest
+import argparse
+import pathlib
+import os
+
+#Note: project_path and engine_path are computed before this script runs, and can be accessed via o3de_context.project_path
+#for now this script assumes an engine-centric approach, and we assume to use the source built engine for the engine_path
+
```

Comment:
I recommend moving this script to under a Platform/Windows folder since it is hardcoded to only work for windows.
There is however a way to generalize it.

The `asset-type` would need to be user configurable, so that they could select between `pc` (Windows), `linux` or `mac` assets

The asset platform option could be an argparse option as well that uses the appropriate default value for the platform.

For example
```python
default_asset_type = 'pc'
if sys.platform == 'win32':
    default_asset_type = 'pc'
elif sys.platform.startswith('linux'):
    default_asset_type = 'linux'
elif sys.platform.startswith('darwin'):
    default_asset_type = 'mac'
else:
    o3de_context.logger.error(f'Export script for platform {sys.platform} does not have a default asset type.' \
        " Using 'pc' as a fallback.")

parser.add_argument('-at, '--asset-type', type=str, default=default_asset_type)
```

Also with the `export_project.py` script, I recommend passing the logger through the context parameters, so that users can log info, warning and error messages

That is just a simple change of adding a logger property to the `O3DEScriptExportContext`
```python
def _run_export_script(args: argparse, passthru_args: list) -> int:
    logging.basicConfig(format=utils.LOG_FORMAT)
    logger = logging.getLogger()
    logger.setLevel(args.log_level)

    return _export_script(args.export_script, args.project_path, passthru_args, logger)
```

Reply:
in case there are hidden gotchas with porting to linux, I'll opt for moving to a platform specific folder for windows

Reply:
also we do provide the logger, but that is passed in to the user's script as `o3de_logger`:
https://github.com/o3de/o3de/blob/7b4fe7cd8ce0341376413283f15a4c40e08361ec/scripts/o3de/o3de/export_project.py#L82

Reply:
I didn't know the user script had access to the logger. That is good.

The spacing is however inconsistent with that call to load_and_execute_script. There is whitespace around the equal sign of the `o3de_context` argument but not around the `o3de_logger` argument.
```python
    return utils.load_and_execute_script(target_script_path, o3de_context=o3de_context, o3de_logger=logging.getLogger())
```